### PR TITLE
Reconcile Documentation updates

### PR DESCRIPTION
**Pull request by vijaypavann-db**
* Formatted `aggregates JSON` configs and added `aggregates` in `reconcile configs`

### CODE REVIEW

File: docs/recon_configurations/README.md [hunk 1]
```diff
@@ -108,6 +109,15 @@ class Table:
 {
   "source_name": "&lt;SOURCE_NAME&gt",
   "target_name": "&lt;TARGET_NAME&gt",
+  "aggregates": [{
```
Comment (sundarshankar89) [inline docs/recon_configurations/README.md:112]: Move this to different table like how we have explained `jdbc_reader_options` etc.

**Review by sundarshankar89 — CHANGES_REQUESTED** [review]

File: docs/recon_configurations/README.md (hunk 1 above)
Comment (vijaypavann-db) [reply docs/recon_configurations/README.md]: Done

**Review by sundarshankar89 — APPROVED** [review]
LGTM

**Review by sundarshankar89 — COMMENTED** [review]
LGTM

**Pull request merged**